tools/diagnostics/service_bundle.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import os
+import retrying
+
+import sdk_cmd
+import sdk_diag
+from sdk_utils import groupby
+
+from diagnostics.bundle import Bundle
+import diagnostics.agent as agent
+
+log = logging.getLogger(__name__)
+
+DEFAULT_RETRY_WAIT = 1000
+DEFAULT_RETRY_MAX_ATTEMPTS = 5
+
+
+class ServiceBundle(Bundle):
+    DOWNLOAD_FILES_WITH_PATTERNS = ["^stdout(\.\d+)?$", "^stderr(\.\d+)?$"]
```

Comment:
I believe these are called something else in sdk 0.50+, please check and amend.

Reply:
Looking at the task sandbox for Kubernetes tasks (which run on SDK 0.50+) it looks like the log files are still called `stdout.N` and `stderr.N`. Do you remember where you saw the thing that gave you that impression?

Reply:
@nickbp said so here: https://jira.mesosphere.com/browse/DCOS-40654?focusedCommentId=171361&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-171361

Reply:
Looks like it's in master only so far? Still, worth preparing for that.

Reply:
33f4e940b80cc8226c494ccaca996aa2fe593a70